tests/installation/accept_license.pm

```diff
@@ -40,6 +40,7 @@ sub run {
         return send_key $cmd{next} unless match_has_tag('license-agreement');
     }
     $self->verify_license_has_to_be_accepted;
+    $self->verify_translation;
```

Comment:
So this is called in every scenario and also for older products? I assume this will break QAM tests as well as potentially openSUSE. I recommend to only enable this check - as well as the previous one - in only selected scenarios so that we do not need to do this in every single scenario but only in selected ones. How about a test variable "INSTALLER_EXTENDED_TEST"?